### PR TITLE
[Communities.gov.uk] Remove broken target (fix #17845)

### LINKED ISSUE #17845: data.gov.uk - https everywhere breaks some download links
Type: ruleset/website issue

Domain: https://data.gov.uk/dataset/d7fcc345-6028-4266-836c-1d7cc6b034c5/english-local-authority-green-belt-dataset

Trying to download data from UK government's website. Data in question is titled "2017-18 Boundaries - Geoserver shapefile download". When HTTPS Everywhere is on, it returns an error:

> Not Found
> 
> The requested URL /geoserver/dclg_inspire/ows was not found on this server.
> 
> Additionally, a 404 Not Found error was encountered while trying to use an ErrorDocument to handle the request. 

If I turn HTTPS Everywhere off just for this site, I still receive the same error. Only after I disable HTTPS Everywhere completely then I can download the data. Other links to data on different subpages of data.gov.uk work fine. Thanks.

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by pipboy96** [comment]
Can you add reason for breakage in comment?

**Comment by Bisaloo** [comment]
@pipboy96, I'm not sure what you mean. The reason is status code mismatch (`HTTP 200` over http and `HTTP 404` over https), as mentioned above.

What do you have in mind?

File: src/chrome/content/rules/communities.gov.uk.xml [hunk 1]
```diff
@@ -32,6 +32,8 @@
 		Status code mismatch:
 		- mcis.erdf.communities.gov.uk
 		- reporting.erdf.communities.gov.uk
+		- maps.communities.gov.uk
+			<test url="http://maps.communities.gov.uk/geoserver/dclg_inspire/ows?service=WFS&version=2.0.0&request=GetFeature&typeName=dclg_inspire:Local_Authority_Greenbelt_boundaries_2017-18&outputFormat=shape-zip&srsName=EPSG:4326" />
```
Comment (pipboy96) [inline src/chrome/content/rules/communities.gov.uk.xml]: not sure about this line

Comment (Bisaloo) [reply src/chrome/content/rules/communities.gov.uk.xml]: It's the example of a page where you can see that the domain is broken. What would you rather have me do?

**Comment by Bisaloo** [comment]
Also please don't push directly on my active branches :confused:. I allow edits but only as a safety in case I'm absent for a long time and someone needs to take over my PRs or if I don't answer and they need to be merge urgently.

File: src/chrome/content/rules/communities.gov.uk.xml (hunk 1 above)
Comment (pipboy96) [reply src/chrome/content/rules/communities.gov.uk.xml]: `Example URL: http://maps.communities.gov.uk/geoserver/dclg_inspire/ows?service=WFS&version=2.0.0&request=GetFeature&typeName=dclg_inspire:Local_Authority_Greenbelt_boundaries_2017-18&outputFormat=shape-zip&srsName=EPSG:4326` would make more sense.

**Pull request merged**